src/common.h

```diff
@@ -122,7 +122,7 @@ extern const uint8_t I2C_ADDR_XCVR;
 /*
  * Time management
  */
-void delay(uint16_t n);
```

Comment:
This was left in as an artifact of my earlier experimentation trying to make a configurable delay for people using RF switches that were triggered by PTT, and is not relevant to this PR.